core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -22,22 +22,38 @@
 import java.util.BitSet;
 
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
-import org.apache.carbondata.core.datastore.page.statistics.ColumnPageStatistics;
```

Comment:
I think we should unify KeyColumnPage and ColumnPage or atleast one is derived from another.